test/unit/specs/components/v-link.spec.js

```diff
@@ -0,0 +1,60 @@
+import { render, screen } from '@testing-library/vue'
+import { warn } from '~/utils/warn'
+import VLink from '~/components/VLink.vue'
+
+jest.mock('~/utils/warn', () => ({ warn: jest.fn() }))
+
+const nuxtLinkStub = {
+  template: '<a v-on="$listeners"><slot /></a>',
+  props: ['to'],
+}
+const nuxtContextMock = {
+  $nuxt: { context: { app: { localePath: (v) => v } } },
```

Comment:
If we change `localePath` to a `jest.fn()` we could test that it get's called with the `to` value for internal links as well.